### ISSUE TITLE
Anyone using our API can access all data 

### ISSUE THREAD

**Issue by robpedersendev** [Lambda-School-Labs/wedding-bells-be#33]
After any user has been logged in to our site, they can perform CRUD operations on our database using our API on live information

**Comment by tinomen** [comment]
What you mean is that they are not scoped to data they own?

**Comment by robpedersendev** [comment]
Yes, thats correct. We are currently working on an admin only middleware in addition to a logged in users middleware

**Comment by tinomen** [comment]
That won't fix it per se. Your queries need to be scoped to the user or account. But don't fret it too much atm. It's a common issue experienced in labs and won't stop you from moving forward.
